docs/root/version_history/v1.16.0.rst

```diff
@@ -5,7 +5,7 @@ Incompatible Behavior Changes
 -----------------------------
 *Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
 
-* build: added visibility rules for upstream. If these cause visibility related breakage, see notes in :repo:`BUILD <v1.16:BUILD>`.
+* build: added visibility rules for upstream. If these cause visibility related breakage, see notes in :repo:`BUILD <https://github.com/envoyproxy/envoy/blob/release/v1.16/BUILD>`.
```

Comment:
im trying to rem how the `repo` tag works - i think it automatically expands the url, but maybe it needs to be hardcoded here, in which case im wondering about using the `repo` tag for this

Reply:
@daixiang0 i think you also need to change `:repo:` -> `:ref:`
checking the rendered link here:

https://storage.googleapis.com/envoy-pr/2882a25/docs/version_history/v1.16.0.html

and it points to:

https://github.com/envoyproxy/envoy/blob/2882a2522a381f45139f942ec2144fd4edc8ee8f/https://github.com/envoyproxy/envoy/blob/release/v1.16/BUILD

Reply:
Done.

@phlax thanks for review.